src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
+
+  min-width: 0;
+  padding: 0;
+
+  width: 40px;
+  height: 40px;
+
+  line-height: 24px;
+  border-radius: 50%;
+
+  &.md-button-focus {
+    background-color: md-color($md-foreground, base, 0.12);
+    @include md-button-theme('background-color', 0.12);
+  }
+
+  .md-button-wrapper > * {
```

Comment:
This is needed to align the first level child items of the wrapper in the middle (correctly to the line-height)